frontend/packages/console-shared/src/components/catalog/catalog-view/CatalogTypeSelector.tsx

```diff
@@ -51,7 +54,32 @@ const CatalogTypeSelector: React.FC<CatalogTypeSelectorProps> = ({
 
           return typeCount > 0 ? (
             <li key={value} className="vertical-tabs-pf-tab" data-test={`tab ${value}`}>
-              <Link to={to}>{`${label} (${typeCount})`}</Link>
+              <Link to={to}>
+                {`${label} (${typeCount})`}{' '}
+                {type.catalogTypeError && (
+                  <Tooltip
+                    content={
+                      type.catalogTypeError instanceof Error ? type.catalogTypeError.message : null
+                    }
+                  >
+                    <YellowExclamationTriangleIcon />
+                  </Tooltip>
+                )}
+              </Link>
+            </li>
+          ) : typeCount === 0 && type.catalogTypeError ? (
```

Comment:
Both conditions are almost the same or?

Couldn't we build it like this:

```tsx
if (typeCount === 0 && !type.catalogTypeError) return null;

return (
  <li>
    <Link to={to}>{`${label} (${typeCount})`}</Link>
    {type.catalogTypeError && (
      <>
        {' '}
        <Tooltip content={
          typeCount === 0 ?
            t() :
            type.catalogTypeError?.message
        }>
          <YellowExclamationTriangleIcon />
        </Tooltip>
      </>
    )}
  </li>
)
```

Reply:
the labels with 0 count should not be a link